.circleci/config.yml

```diff
@@ -0,0 +1,105 @@
+version: 2.1
+executors:
+  default-executor:
+    docker:
+      - image: "cimg/node:12.22.11-browsers"
+    working_directory: ~/project
+    resource_class: medium
+
+commands:
+  restore_test_cache:
+    steps:
+      - restore_cache:
+          keys:
+            - v1-deps-{{ checksum "package-lock.json" }}
+            - v1-deps-
+  save_test_cache:
+    steps:
+      - save_cache:
+          key: v1-deps-{{ checksum "package-lock.json" }}
+          paths:
+            - node_modules
+            - ~/.cache/ms-playwright
+
+  setup:
+    steps:
+      - run:
+          name: Install dependencies
+          command: |
+            npm --production=false install
+            npm --production=false update
+            npx playwright install
```

Comment:
Could you explain what the `npx playwright install` does here? I don't see it in the old Travis script.

Reply:
I believe it installs playwright for testing. I added this because circle was complaining and telling me to do so.